### PR TITLE
det-1693: technique in template

### PR DESCRIPTION
**Pull request by packplusplus**
~Techniques are required in tests now, require on CLI, and~ document technique in the template.

### CODE REVIEW

File: python/cli/prelude_cli/views/build.py [hunk 1]
```diff
@@ -41,6 +41,7 @@ def create_template(template, name):
         template_body = template_body.replace('$ID', t['id'])
         template_body = template_body.replace('$NAME', t['name'])
         template_body = template_body.replace('$UNIT', t['unit'])
+        template_body = template_body.replace('$TECHNIQUE', t['technique'])
```
Comment (makk94) [inline python/cli/prelude_cli/views/build.py:44]: think this'll error if technique is none. might need to `str(t['technique'])` or smth

File: python/cli/prelude_cli/views/build.py [hunk 2]
```diff
@@ -36,12 +36,12 @@ def build(ctx):
...
-        utc_time = str(datetime.now(timezone.utc))
         template_body = pkg_resources.read_text(templates, template)
-        template_body = template_body.replace('$ID', t['id'])
-        template_body = template_body.replace('$NAME', t['name'])
-        template_body = template_body.replace('$UNIT', t['unit'])
-        template_body = template_body.replace('$TIME', utc_time)
+        template_body = template_body.replace('$ID', t.get('id'))
+        template_body = template_body.replace('$NAME', t.get('name'))
+        template_body = template_body.replace('$UNIT', t.get('unit'))
+        template_body = template_body.replace('$TECHNIQUE', t.get('technique'))
```
Comment (makk94) [inline python/cli/prelude_cli/views/build.py:43]: t.get('technique') will still return None and its the text.replace('blah', None) that errors

i'd go back to the direct `t['blah']` format - the .get()s don't get you anything i dont think

Comment (packplusplus) [reply python/cli/prelude_cli/views/build.py]: I misunderstood, in two different ways. I think we want an empty value if there's no technique, not `None` or `''`.

**Review by makk94 — APPROVED** [review]

**Comment by makk94** [comment]
nits: you could update the sdk test templates too but that'd be more for ultra consistency than any actual use
https://github.com/search?q=repo%3Apreludeorg%2Flibraries+%24UNIT&type=code

**Comment by packplusplus** [comment]
> nit: you could update the sdk test templates too but that'd be more for ultra consistency than any actual use https://github.com/search?q=repo%3Apreludeorg%2Flibraries+%24UNIT&type=code

nits are good, but the javascript sdks starts platform. so pr there

**Review by makk94 — APPROVED** [review]

File: python/sdk/tests/templates/template.go [hunk 3]
```diff
@@ -1,21 +1,22 @@
...
 package main
 
 import (
-	"github.com/preludeorg/libraries/go/tests/endpoint"
+	Endpoint "github.com/preludeorg/libraries/go/tests/endpoint"
 )
 
 func test() {
-	Endpoint.Stop(100)
+	Endpoint.Stop(Endpoint.TestCompletedNormally)
```
Comment (makk94) [inline python/sdk/tests/templates/template.go:15]: bueno 👍

**Pull request merged**